### PR TITLE
Bad usage of `tax_query` param deletes a lot more than it should

### PR DESCRIPTION
**Pull request by aidvu**
## Summary

Wrong usage of `tax_query`. Please check: https://developer.wordpress.org/reference/classes/wp_query/#taxonomy-parameters. This ended up deleting all reusable blocks for our customers. A hotfix release would be good.

Fixes: https://github.com/ampproject/amp-wp/issues/4489

## Checklist

- [x] My pull request is addressing an [open issue](https://github.com/ampproject/amp-wp/contributing/project-management.md#life-of-an-issue) (please create one otherwise).
- [ ] My code is tested and passes existing [tests](https://github.com/ampproject/amp-wp/contributing/engineering.md#tests).
- [ ] My code follows the [Engineering Guidelines](https://github.com/ampproject/amp-wp/contributing/engineering.md) (updates are often made to the guidelines, check it out periodically).

### CODE REVIEW

**Comment by aidvu** [comment]
@googlebot I signed it!

**Comment by aidvu** [comment]
Thank you @westonruter. If possible, I'd suggest pushing a new `readme.txt` with a previous version in the `stable` tag. This is going to cause a lot of mess...

**Comment by westonruter** [comment]
> If possible, I'd suggest pushing a new `readme.txt` with a previous version in the `stable` tag. This is going to cause a lot of mess...

I don't understand.

**Comment by aidvu** [comment]
> I don't understand.

I'm seeing this as a major bug, and would suggest reverting the stable version back to 1.4.4 (https://github.com/ampproject/amp-wp/blob/develop/readme.txt#L6), and then working on a 1.5.1 release.

**Comment by westonruter** [comment]
We'll release 1.5.1 within the hour.

**Comment by aidvu** [comment]
> We'll release 1.5.1 within the hour.

Thank you for the fast responses and working on this. 🙇

File: includes/admin/functions.php [hunk 1]
```diff
@@ -209,9 +209,11 @@ function remove_amp_story_templates() {
...
 				'post_per_page' => -1,
 				'tax_query'     => [
-					'taxonomy' => $template_taxonomy,
-					'field'    => 'slug',
-					'terms'    => [ $template_term ],
+					[
+						'taxonomy' => $template_taxonomy,
+						'field'    => 'slug',
+						'terms'    => [ $template_term ],
+					],
```
Comment (westonruter) [inline includes/admin/functions.php]: I'm finding this change is resulting in no story template blocks being removed at all. Investigating…

**Comment by aidvu** [comment]
Just wondering, couldn't get it to do what you wanted, or it's better not to delete users posts (and I agree)?

**Comment by westonruter** [comment]
I wasn't able to reliably reproduce the issue and for the sake of time and hotfix urgency, better to just remove for now. The worst case is the reusable blocks will include the list of the templates:

image

But they can just be deleted by the user.

**Pull request merged**